assignments/objects.js

```diff
@@ -49,16 +91,40 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  name: "Susan",
+  age: 70,
+  speak: function() {
+    console.log( "My name is " + this.name + "!")
+  },
+
+  child: {
+    name: "George",
+    age: 50,
+    speak: "My name is George!",
+
+    grandchild: {
+      name: "Sam",
+      age: 30,
+      speak: "My name is Sam!"
```

Comment:
in the child and grandchild objects "speak" is not a method, it's a string. Can you change these into methods?